tst/regression/mocmc_diffusion.gold

```diff
@@ -1,76 +1,25 @@
-1.107667635301233942e-06
-8.678469590435074669e-11
-9.971481965236875367e-11
-9.639947382410148605e-09
-1.268340826133416413e-03
-6.562967553167058377e-01
-4.470716447853096201e-04
-9.999999999999995194e-11
-9.999999999999995194e-11
-2.334632208492531946e-02
-7.622557034077365179e-02
-9.999999999999995194e-11
-9.999999999999995194e-11
-9.999999999999995194e-11
-9.999999999999995194e-11
-1.495235949844014333e-05
-9.927199198898888355e-01
-5.498364121645113345e-07
-9.999999999999995194e-11
-9.999999999999995194e-11
-5.716019338297739566e-04
--2.825591164626063668e-04
--2.420653647569474473e-02
--2.049083258733951779e-02
--7.560785253149245912e-03
-2.678448152425867007e-02
-1.232937520260024407e-16
-1.232937520260024407e-16
-1.090170239953292955e-16
-1.717855834510626225e-16
-8.487991656731448068e-17
-1.232937520260024407e-16
-1.232937520260024407e-16
-1.232937520260024407e-16
-1.358067774462864810e-16
-6.886394415575321099e-17
-0.000000000000000000e+00
-1.232937520260024407e-16
-1.232937520260024407e-16
-4.909950132900829226e-19
--2.081219149297657992e-18
--3.534472263881267203e-18
-3.058232911759794560e-19
--6.143732111690850347e-19
--2.095180251051548739e-18
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-3.720535392342713946e-18
-4.275850048651767063e-18
--5.382624650889726464e-19
--7.417127987768036823e-20
-4.497843815824792516e-18
--1.111063243719671523e-18
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
-0.000000000000000000e+00
+2.578814482748378809e-07
+8.539726882830999535e-11
+9.881492890533366489e-11
+2.878000679792632022e-08
+2.198127933800966046e-03
+6.512936284317737412e-01
+1.589897082372886724e-04
+2.487714603868343421e-02
+2.197674234395436202e-02
+1.035157783173219770e-02
+2.531339100900154189e-03
+1.082842734053430199e-02
+4.744884335341734150e-02
+4.023128250369822722e-18
+-4.074391331036127547e-18
+-1.188232745820429572e-17
+-4.992172249544863943e-20
+3.600838140944280810e-20
+3.223026185159611409e-18
+3.207937073915787390e-18
+-1.678974299604244268e-18
+4.709490397007393027e-19
+-3.950108603179772350e-18
+-2.606358899989489921e-19
+1.986847194014894714e-19
```

Comment:
These gold files shrunk because we aren't always outputting 3 rad species of data any more